tasks/eth/013-fp-upgrade-fjord/VALIDATION.md

```diff
@@ -0,0 +1,137 @@
+
+### `0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A` (The 2/2 `ProxyAdmin` Owner)
+
+State Changes:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005`
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000003`
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000004`
+  **Meaning:** The nonce is increased from 4 to 5. The key can be validated by the location of the nonce variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L17).
+
+#### For the Council:
+
+- **Key:** `0x0f7b79f6b38abe5e02d33eec6fcdf7d9447ff17f6803d46f0afd5f628bac8504` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000`<br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** The GnosisSafe `approvedHashes` mapping is updated to indicate approval of this transaction by the council. The correctness of this slot can be verified as follows:
+    - Since this is a nested mapping, we need to use `cast index` twice to confirm that this is the correct slot. The inputs needed are:
+      - The location (`8`) of the `approvedHashes` mapping in the [GnosisSafe storage layout](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L20)
+      - The address of the Council Safe: `0xc2819DC788505Aac350142A7A707BF9D03E3Bd03`
+      - The safe hash to approve: `0x530085da61c08016f1625301df4686fcfe40291b98db29a4bf801237ca3098c4`
+    - The using `cast index`, we can verify that:
+      ```shell
+        $ cast index address 0xc2819DC788505Aac350142A7A707BF9D03E3Bd03 8
+        0xaaf2b641eaf0bae063c4f2e5670f905e1fb7334436b902d1d880b05bd6228fbd
+        ```
+        and
+      ```shell
+        $ cast index bytes32 0x530085da61c08016f1625301df4686fcfe40291b98db29a4bf801237ca3098c4 0xaaf2b641eaf0bae063c4f2e5670f905e1fb7334436b902d1d880b05bd6228fbd
+        0x0f7b79f6b38abe5e02d33eec6fcdf7d9447ff17f6803d46f0afd5f628bac8504 
+        ```
+      And so the output of the second command matches the key above.
+
+- **Key:** `0x24424336f04440b1c28685a38303ac33c9d14a25`
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000`
+  **After:** `0x000000000000000000000000000000000000000000000000000000006675b61f`
+  **Meaning:** This updates the [`lastLive`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.5.0/packages/contracts-bedrock/src/Safe/LivenessGuard.sol#L36) indicating liveness of an owner that participated in signing. This will be updated to a block timestamp that's close to when this task was executed.
```

Comment:
I think you are missing the header here since this is the Liveness Guard address, not the 2/2 Proxy Admin Owner. Also, the before value may not be zero for some signers—we can just note that in the Meaning text

Reply:
Good catch. Thanks for your patience 😅